style.css

```diff
@@ -2609,6 +2609,28 @@ article.panel-placeholder {
 		font-size: 16px;
 		font-size: 1.0rem;
 	}
+
+	/* Gallery Columns */
+	
+	.gallery-item {
+		max-width: 25%;
+	}
+
+	.gallery-columns-1 .gallery-item {
+		max-width: 100%;
+	}
+
+	.gallery-columns-2 .gallery-item {
+		max-width: 50%;
+	}
+
+	.gallery-columns-3 .gallery-item {
+		max-width: 33.33%;
+	}
+
+	.gallery-columns-4 .gallery-item {
```

Comment:
This is redundant, since `.gallery-item` is already `max-width: 25%`, but might be good to leave for clarity? I'm not sure.